Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -444,34 +458,34 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
         auto boundingRect = absoluteBoundingRect(regionRenderer);
         clipPath = styleClipPath->getPath(TransformOperationData(FloatRect(FloatPoint(), boundingRect.size())));
     } else if (iconImage && originalElement) {
-        LayoutRect imageRect(rect);
+        auto boundingRect = absoluteBoundingRect(regionRenderer);
```

Comment:
Calling `absoluteBoundingRect` just to get its size is wasteful; it pays the cost for mapping the rect to absolute coordinates.

Reply:
we use the location in `pathTranslationToMatchClipping` below

Reply:
we still need the transform but this should be better